lib/matplotlib/backends/backend_tkagg.py

```diff
@@ -1,17 +1,15 @@
-from . import _backend_tk
 from .backend_agg import FigureCanvasAgg
 from ._backend_tk import (
-    _BackendTk, FigureCanvasTk, FigureManagerTk, NavigationToolbar2Tk)
```

Comment:
I think you can keep the old import pattern?  This avoids making `blit` public (via `backend_tkagg.blit`).

Reply:
fine by me